dist/backdraft.js

```diff
@@ -845,11 +845,19 @@ _.extend(Plugin.factory, {
     ',
     sFilterListTemplate : '\
       <li><label><input class="list" id="value" type="checkbox" name="{{= attr }}" \
-         value="{{= options }}" /> <%= options %></label></li> \
+         value="{{= options }}" /> {{= options }}</label></li> \
+    ',
+    sActionButtonTemplate : '\
+      <button class="btn btn-primary btn-sm {{= klass}}" name="button" type="submit" title="">{{= caption}}</button>\
+    ',
+    sFilterButtonTemplate : '\
```

Comment:
is this the icon that toggles the filter UI?  I agree this name is too general. Assuming you nest it in the filterTemplates hash it could be toggleButton or something that describes what this item is doing